src/1-Profile/Settings.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, TextStyle, Text, Modal, Linking } from 'react-native';
+import theme, { FONT_SIZES } from '../theme';
+import { Outline_Button, Raised_Button } from '../widgets';
+import { StackNavigationProps } from '../TypesAndInterfaces/custom-types';
+import { ROUTE_NAMES } from '../TypesAndInterfaces/routes';
+import { PARTNERSHIP_CONTRACT } from '../TypesAndInterfaces/config-sync/input-config-sync/profile-field-config';
+import Partnerships from '../4-Partners/Partnerships';
+import { DOMAIN } from '@env';
+import axios, { AxiosError } from 'axios';
+import { useAppSelector } from '../TypesAndInterfaces/hooks';
+import { RootState } from '../redux-store';
+import ToastQueueManager from '../utilities/ToastQueueManager';
+import { ServerErrorResponse } from '../TypesAndInterfaces/config-sync/api-type-sync/toast-types';
+
+const ProfileSettings = ({navigation}:StackNavigationProps):JSX.Element => {
+
+    const jwt = useAppSelector((state: RootState) => state.account.jwt);
+    const [partnerModalVisible, setPartnerModalVisible] = useState(false);
+    const RequestAccountHeader = {
+        headers: {
+          "jwt": jwt, 
+        }
+      }
+
+    const onLogout = () => {
+        axios.post(`${DOMAIN}/api/logout`, {}, RequestAccountHeader).then((response) => {
```

Comment:
Can we clear redux jwt and profile; then redirect back to login page? This would be helpful while debugging.

Since the server is not storing sessions; maybe its not necessary to notify the server on logout.  I suppose once we have persistent authentication, a true logout will be rare and maybe that's worth notifying the server, IDK.

Reply:
Makes sense. Updated to only clear redux without contacting the server. navigation's popToTop() pops the navigation screen stack to the first screen defined, which for us is the login page.